testsuite/.rubocop_todo.yml

```diff
@@ -1,427 +1,25 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2023-02-24 14:47:24 +0100 using RuboCop version 0.83.0.
+# on 2023-11-17 13:14:03 UTC using RuboCop version 1.28.0.
```

Comment:
Shouldn't this be created with RubyCop version 1.17?

Reply:
Now that surprised me, it has been all the time Rubocop 1.17.0 and suddenly I have setup Rubocop 1.28.0.
Not sure how that happened... but I fixed in a new commit.